scripts/containers.sh

```diff
@@ -7,19 +7,19 @@ APP_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )/"
 cd $APP_HOME
 
 start() {
-    ./scripts/pcsconfig.sh start
     ./scripts/storageadapter.sh start
     ./scripts/devicesimulation.sh start
     ./scripts/telemetry.sh start
+    ./scripts/config.sh start
```

Comment:
is there any issue with moving config down?

Reply:
Config needs to be started after the containers above it, in order to successfully complete seeding on startup. Though we have 10 retry with 10 sec interval for seeding, it's better this way.